utils/prometheus/README.md

```diff
@@ -0,0 +1,555 @@
+				  P2P_NODE_DOWNLOAD => net_status.average_download_per_sec as u64,
+				  P2P_NODE_UPLOAD => net_status.average_upload_per_sec as u64
+				);
+.....
+```
+## Metrics
+
+substrate can report and serve the Prometheus metrics, which in their turn can be consumed by Prometheus collector(s).
+
+This functionality is disabled by default.
+
+To enable the Prometheus metrics, set in your cli command (--prometheus-addr,--prometheus-port ). 
+Metrics will be served under /metrics on 33333 port by default.
+
+### List of available metrics
+
+
+Consensus metrics, namespace: `substrate`
+
+| **Name**                               | **Type**  | **Tags** | **Description**                                                 |
+| -------------------------------------- | --------- | -------- | --------------------------------------------------------------- |
+| consensus_finality_block_height_number | IntGauge  |          | finality Height of the chain                                    |
+| consensus_best_block_height_number     | IntGauge  |          | best Height of the chain                                        |
+| consensus_target_syn_number            | IntGauge  |          | syning Height target number                                     |
+| consensus_num_txs                      | Gauge     |          | Number of transactions                                          |
+| consensus_node_memory                  | IntGauge  |          | Node's primary memory                                           |
+| consensus_node_cpu                     | IntGauge  |          | Node's cpu load                                                 |
+| consensus_state_cache_size             | IntGauge  |          | used state cache size                             			  |
+| p2p_peers_number                       | IntGauge  |          | Number of peers node's connected to                             |
+| p2p_peer_receive_bytes_per_sec         | IntGauge  |          | number of bytes received from a given peer                      |
+| p2p_peer_send_bytes_per_sec            | IntGauge  |          | number of bytes sent to a given peer                            |
+| Resource_receive_bytes_per_sec(Future) | IntGauge  |          | Operating System of bytes received                              |
+| Resource_send_bytes_per_sec(Future)    | IntGauge  |          | Operating System of bytes sent                                  |
+| Resource_cpu_use(Future)               | IntGauge  |          | Operating System cpu load                                       |
+| Resource_disk_use(Future)              | IntGauge  |          | Operating System disk use                                       |
+| validator_sign_prevote(Future)         | IntGauge  | validator addr | validator sign vote list                               	  |
+| validator_sign_precommit(Future)       | IntGauge  | validator addr | validator sign commit list                                |
+
+
+## Start Prometheus
```

Comment:
I don't think this is worth duplicating here. I would suggest to link to the Prometheus docs.

Reply:
I agree. I'm dropping the table

Reply:
complete